index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio || BB Simon</title>
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="./styles/style.css">
+
+  <!-- Google fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap"
+    rel="stylesheet">
+
+</head>
+<body>
+  <!-- Navigation bar -->
+  <nav class="navbar">
+    <ul class="nav-items-wrapper">
+      <li class="nav-item">
+        <a class="logo-wrapper" href="/">
+          <img src="./assets/My Logo.png" alt="Logo">
+        </a>
+      </li>
+      <li class="nav-item">
+        <img src="./assets/Icon - Menu.png" alt="Menu">
+      </li>
+    </ul>
+  </nav>
```

Comment:
:red_circle: [REQUIRED] Please make sure the parent container of this section is a header tag. :heavy_plus_sign: Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
This portion of your code should look as in this example :arrow_down: 
```
<header>
   <nav>
      <!-- Add the nav content here -->
  </nav>
</header>
```